FrontEnd/DevPaths/src/components/Footer/index.tsx

```diff
@@ -0,0 +1,55 @@
+import React from "react";
+import "./Footer.module.scss";
```

Comment:
```suggestion
import style from "./Footer.module.scss";
```
import ` scss ` files as a module.